cardano-cli/src/Cardano/CLI/Byron/Commands.hs

```diff
@@ -87,6 +88,7 @@ data ByronCommand =
   | SpendUTxO
         NetworkId
         ByronKeyFormat
+        ScriptValidity
```

Comment:
Why are we modifying the Byron era? There are no scripts in the Byron era.

Reply:
Fixed